category/vm/evm/opcodes.hpp

```diff
@@ -69,7 +69,7 @@ namespace monad::vm::compiler
          * Some instructions may also consume additional dynamic gas depending
          * on run-time properties (e.g. memory expansion or storage costs).
          */
-        std::uint16_t min_gas;
+        std::int32_t min_gas;
```

Comment:
Needs to change as 160k > max uint32. Using int32 as gas is tracked as int32 in the VM anyway